app/models/import/row.rb

```diff
@@ -81,4 +84,20 @@ def currency_is_valid
         errors.add(:currency, "is not a valid currency code")
       end
     end
+
+    def exchange_operating_mic_is_valid
+      return true if exchange_operating_mic.blank?
+      return true unless Security.security_prices_provider.present?
+
+      response = Security.security_prices_provider.fetch_security_prices(
+        ticker: ticker,
+        mic_code: exchange_operating_mic,
+        start_date: Date.current,
+        end_date: Date.current
+      )
+
+      unless response.success?
+        errors.add(:exchange_operating_mic, "is not valid for ticker #{ticker}. No prices found for this combination.")
+      end
+    end
```

Comment:
I'm second guessing this.  I know I said in the requirements that we need a per-cell validation, but I think our best bet is to remove this and check for tickers directly during the import operation.

This will make a Synth call for every single row in the import, which could quickly exhaust a user's Synth credits.  I think I'd rather just remove this for now.